cmd/notify-nats.go

```diff
@@ -52,7 +53,7 @@ func (n *natsNotify) Validate() error {
 	if !n.Enable {
 		return nil
 	}
-	if _, err := checkURL(n.Address); err != nil {
+	if _, _, err := net.SplitHostPort(n.Address); err != nil {
```

Comment:
@harshavardhana we can have default ports for the service if port is not specified as opposed to failing due to `net.SplitHostPort`'s requirement right? Why penalise the user for library quirks?